faster_whisper/transcribe.py

```diff
@@ -227,10 +225,7 @@ def audio_split(audio, segments, sampling_rate):
 
     def transcribe(
         self,
-        audio: Union[str, torch.Tensor, np.ndarray],
-        vad_segments: Optional[List[dict]] = None,
```

Comment:
Wouldn't it be possible to optionally pass in the `vad_segments` to support any external VAD models as needed? This can default to `None` to make it look as close as a sequential transcribe. This is beneficial if you have multiple transcription requests and the framework can process VAD of next audio while doing transcription of current audio.

Reply:
This is already possible using `clip_timestamps`, why did I remove `vad_segments` and added `clip_timestamps`? I'm trying to make `transcribe` functions in both batched and sequential as close as possible and interchangeable